rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -46,7 +46,7 @@ memory_viewer_panel::memory_viewer_panel(QWidget* parent)
 	QGroupBox* tools_mem_addr = new QGroupBox(tr("Address"));
 	QHBoxLayout* hbox_tools_mem_addr = new QHBoxLayout();
 	m_addr_line = new QLineEdit(this);
-	m_addr_line->setPlaceholderText("00000000");
+	m_addr_line->setText(qstr(fmt::format("%x", m_addr)));
```

Comment:
Why remove the placeholder text. It has nothing to do with the text

Reply:
Fixed.